addons/common/XEH_PREP.hpp

```diff
@@ -46,6 +46,7 @@ PREP(fixCollision);
 PREP(fixFloating);
 PREP(fixLoweredRifleAnimation);
 PREP(fixPosition);
+PREP(garrison);
```

Comment:
Doesn't CBA now have garrison functions that could be used instead? If not, then yeah, those should go into `ace_ai` component.